src/bitcoind_client.rs

```diff
@@ -163,18 +167,28 @@ impl BitcoindClient {
 					}
 				};
 
-				fees.get(&ConfirmationTarget::MempoolMinimum)
+				fees.get(&ConfirmationTarget::OnChainSweep)
+					.unwrap()
+					.store(high_prio_estimate, Ordering::Release);
+				fees.get(&ConfirmationTarget::MaxAllowedNonAnchorChannelRemoteFee)
+					.unwrap()
+					.store(std::cmp::max(25*250, high_prio_estimate), Ordering::Release);
```

Comment:
@TheBlueMatt should this be: 

`max(25*250, high_prio_estimate * 10)`

That seems to be the guidance in the release notes: https://github.com/lightningdevkit/rust-lightning/releases/tag/v0.0.118